### DISCUSSION TITLE
Example of Watch

### DISCUSSION THREAD

**Discussion by SadykovR** [etcd-io/jetcd#1070] [Q&A]
Can you share an example about how to watch key with etcd? Thank you!
I use 0.5.0 version.
I found a discussion with the same question https://github.com/etcd-io/jetcd/issues/853.
And the solution is not found yet.
So what I am supposed to do?

**Comment by lburgazzoli** [comment]
There are a bunch of examples in the test suite, as example: https://github.com/etcd-io/jetcd/blob/main/jetcd-core/src/test/java/io/etcd/jetcd/impl/WatchTest.java

Is there anything missing ?

Note that if you have conflicts with the underlying grpc/netty library, then you may get subtle and silent errors.

I would recommend to set-up a test with only jetcd as a dependency and to contribute a filing test in case you find something not working.

**Reply by SadykovR** [reply]
Thank you for your reply!
Still doesn't work properly.
I guess we have some issues with ports, maybe some ports are closen.
